File: MagicBricks/API/Source/Magicbrics/Models/Property.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Magicbrics.Models
{
    public partial class Property
    {
        public Property()
        {
            ContactOwners = new HashSet<ContactOwner>();
            Images = new HashSet<Image>();
            Locations = new HashSet<Location>();
            PropertyFacilities = new HashSet<PropertyFacility>();
        }

        public int PropertyId { get; set; }
        public int? PropertyTypeId { get; set; }
        public string PropertyDescription { get; set; }
        public int? BrsId { get; set; }
        public int? UserId { get; set; }

        public virtual ApplicationObjectType Brs { get; set; }
        public virtual ApplicationObjectType PropertyType { get; set; }
        public virtual ICollection<ContactOwner> ContactOwners { get; set; }
        public virtual ICollection<Image> Images { get; set; }
        public virtual ICollection<Location> Locations { get; set; }
        public virtual ICollection<PropertyFacility> PropertyFacilities { get; set; }
    }
}

File: MagicBricks/FrontEnd/Source/src/app/Models/Property.ts
import Images from "./Images";
import Locations from "./Locations";
import PropertyFacility from "./PropertyFacility";

export default interface property{
PropertyID?:number|null;
PropertyTypeID:number|null;
PropertyDescription:string|null;
BrsId:number|null;
UserID:number|null;
Locations?:Locations[]|null;
PropertyFacilities?:PropertyFacility[]|null;
Image?:Images[]|null;
}